parser/src/cfg/sexpr.rs

```diff
@@ -29,14 +29,25 @@ impl Position {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[derive(Clone, PartialEq, Eq, Hash)]
 pub struct Span {
     pub start: Position,
     pub end: Position,
     pub file_name: Rc<str>,
     pub file_content: Rc<str>,
 }
 
+impl Debug for Span {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
```

Comment:
Curiosity:

What's the rationale behind the manual `Debug`? I'm guessing not printing the whole of `file_content`? (does Rc<str> do that?)